content/operate/kubernetes/upgrade/upgrade-redis-cluster.md

```diff
@@ -159,8 +164,11 @@ kubectl rollout status sts <REC_name>
 
 ### Upgrade databases
 
+After the cluster is upgraded, you can upgrade your databases. The process for upgrading databases is the same for both Kubernetes and non-Kubernetes deployments.
+
+For more details on how to [upgrade a database]({{< relref "/operate/rs/installing-upgrading/upgrading/upgrade-database" >}}), see the [Upgrade an existing Redis Enterprise Software deployment]({{< relref "/operate/rs/installing-upgrading/upgrading" >}}) documentation.
 
-After the cluster is upgraded, you can upgrade your databases. The process for upgrading databases is the same for both Kubernetes and non-Kubernetes deployments. For more details on how to [upgrade a database]({{< relref "/operate/rs/installing-upgrading/upgrading/upgrade-database" >}}), see the [Upgrade an existing Redis Enterprise Software deployment]({{< relref "/operate/rs/installing-upgrading/upgrading" >}}) documentation.
+For Active-Active databases, see [Upgrade an Active-Active database]({{<relref "/operate/rs/installing-upgrading/upgrading/upgrade-active-active/">}}).
```

Comment:
For some reason, the current link takes me to https://redis.io/docs/staging/DOC-2573/operate/kubernetes/upgrade/upgrade-redis-cluster/ instead of the RS Active-Active upgrade doc. I don't know if that happens for you too in the staged preview. If so, maybe try removing the trailing slash:
```suggestion
For Active-Active databases, see [Upgrade an Active-Active database]({{<relref "/operate/rs/installing-upgrading/upgrading/upgrade-active-active">}}).
```